package.json

```diff
@@ -48,9 +48,9 @@
     "postinstall:project": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
     "publish": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
     "postpublish": "npm run dependency-markdown && git add --all && git commit -m 'Updated DEPENDENCIES.md files'",
-    "start": "tt:serve",
-    "start-heroku": "tt:serve-static --site build ",
-    "start-static": "tt:serve-static --config webpack.config",
+    "start": "tt-serve",
+    "start-heroku": "tt-serve-static --site build ",
+    "start-static": "tt-serve-static --config webpack.config",
     "test": "npm run jest && npm run wdio",
     "wdio": "wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js",
     "tds:generate-app-config": "tds:generate-app-config"
```

Comment:
This needs to be changed to `tds-generate-app-config` to align with the terra-dev-site update

Reply:
good catch! I bet I missed these in each repo 😭